managed/services/management/dbaas/pxc_cluster_service.go

```diff
@@ -348,7 +348,11 @@ func (s PXCClustersService) UpdatePXCCluster(ctx context.Context, req *dbaasv1be
 	if err != nil {
 		return nil, err
 	}
-	err = kubernetes.UpdatePatchForPXC(dbCluster, req)
+	clusterType, err := kubeClient.GetClusterType(ctx)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed getting cluster type")
+	}
+	err = kubernetes.UpdatePatchForPXC(dbCluster, req, clusterType)
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to create CR specification")
```

Comment:
minor thing here but the operation above is to patch the spec, maybe this error message is inconsistent?